features/css-object-model.yml

```
@@ -0,0 +1,83 @@
+name: CSS Object Model
+description: The CSS Object Model allows users to create and change CSS styles using JavaScript. Also known as CSSOM.
```

A few things here:

- Ugh, this has so many plausible entry points, which makes describing this difficult.
- I don't think we should mention JavaScript, since it's a browser API, not a JS built-in.
- We should probably have a style entry for "users" to only use it to mean something like "the users of the pages/applications that developers make" and not "developers." Or I have badly misunderstood what CSSOM is for.

```suggestion
description: The CSS object model API reads, creates, and modifies CSS stylesheets and inline styles. Also known as CSSOM.
```